lightning/src/routing/router.rs

```diff
@@ -625,6 +625,30 @@ impl PaymentParameters {
 		Self::from_node_id(payee_pubkey, final_cltv_expiry_delta).with_bolt11_features(InvoiceFeatures::for_keysend()).expect("PaymentParameters::from_node_id should always initialize the payee as unblinded")
 	}
 
+	/// Creates parameters for paying to a blinded payee.
+	pub fn blinded(blinded_route_hints: Vec<(BlindedPayInfo, BlindedPath)>) -> Self {
+		Self {
+			payee: Payee::Blinded { route_hints: blinded_route_hints, features: None },
+			expiry_time: None,
```

Comment:
expiry_time `None` because we don't have an invoice to lift these out of yet?

Reply:
Expiry is provided via `PaymentParameters::with_expiry_time` :)